features/steps/events_steps.rb

```diff
@@ -362,7 +362,7 @@ def icecube_schedule_to_time_restriction(sched, time_zone)
 
 Given /^user (\S+) has the following notification rules:$/ do |contact_id, rules|
   contact = Flapjack::Data::Contact.find_by_id(contact_id, :redis => @redis)
-  timezone = contact.timezone
+  time_zone = contact.time_zone
```

Comment:
Yeah, there was some confusion internally in the code prior. 'timezone' as a string should always be the external interface, and `time_zone` is used for the instantiation thereof.